src/sentry/incidents/endpoints/organization_alert_rule_index.py

```diff
@@ -72,7 +76,7 @@ def get(self, request, organization):
                 if team.id in verified_ids:
                     continue
 
-                if not request.access.has_team_access(team):
+                if not request.is_superuser() and not request.access.has_team_access(team):
```

Comment:
I think the `is_superuser` check is unnecessary here, since superusers should have `has_global_access` set to True. Probably worth a quick test to verify.

Reply:
Gotcha. I'll verify

Reply:
You were right